### PR TITLE
Chore: Update Internet Identity Init type

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

User can see the latest "memory_migration_batch_size" argument name used when upgrading the II canister.

# Changes

* Add "memory_migration_batch_size" to InternetIdentityInit hardcoded type.

# Tests

Not applicable, only UI changes.

### CODE REVIEW

**Comment by lmuntaner** [comment]
New UI:

Screenshot 2022-12-05 at 09 33 38

File: rs/src/proposals.rs [hunk 1]
```diff
@@ -73,6 +73,7 @@ struct InternetIdentityInit {
     pub assigned_user_number_range: Option<(UserNumber, UserNumber)>,
     pub archive_module_hash: Option<[u8; 32]>,
     pub canister_creation_cycles_cost: Option<u64>,
+    pub memory_migration_batch_size: Option<u64>,
```
Comment (frederikrothenberger) [inline rs/src/proposals.rs:76]: It is actually an `u32`.

**Comment by frederikrothenberger** [comment]
You can copy the type from here: https://github.com/dfinity/internet-identity/blob/main/src/internet_identity_interface/src/lib.rs#L174

File: rs/src/proposals.rs (hunk 1 above)
Comment (lmuntaner) [reply rs/src/proposals.rs:76]: thanks!

**Review by frederikrothenberger — APPROVED** [review]

**Pull request merged**